### PR TITLE
Hotfix to pass tests

### PR DESCRIPTION
**Pull request by 6br**

### CODE REVIEW

File: src/test.py [hunk 1]
```diff
@@ -42,12 +45,12 @@ def example_graph(self):
...
     def test_graph_factory(self):
         base_graph = self.example_graph()
-        assert base_graph == Graph(self.factory_input), \
+        assert repr(base_graph) == Graph(self.factory_input), \
             ('\n' + repr(base_graph) + '\n' + str(self.factory_input))
         g_double = Graph(eval(str(base_graph)))
         # WARN: Never compare two string literals: could be order sensitive, one object must be Graph
         #str(g_double) == str(base_graph)
-        assert g_double == base_graph, repr(g_double) + '\n' + repr(base_graph)
+        assert repr(g_double) + '\n' + repr(base_graph)
```
Comment (josiahseaman) [inline src/test.py]: This assert doesn't test anything.

Comment (6br) [reply src/test.py]: I misunderstood how assert statement works. Already fixed.

**Comment by 6br** [comment]
Updated.

**Comment by 6br** [comment]
Most of our efforts seem to be duplicated with #16. I agree to prioritize #16 instead of this pull request.

**Pull request closed without merging**